### ISSUE TITLE
Page <title>s need to be populated

### ISSUE THREAD

**Issue by zarino** [mysociety/remit#61]
I'm guessing it's best for the three homepage tabs to be:

* All Studies – ReMIT
* Your Studies – ReMIT
* Studies you’re following – ReMIT

(Or whatever the three tabs end up being called)

Not sure whether we customise the title for search results / filtered results? Could get quite complicated?

* Studies matching "hepatitis e" – ReMIT
* Studies with ERB approval pending – ReMIT
* Studies with publications or registered impacts – ReMIT

Individual studies should probably follow the same pattern, with the study name first – possibly truncated to 50 characters or thereabouts to make them less unwieldy in users's history and bookmark menus?

* Understanding traditional pregnancy, childbirth and… – ReMIT
* MSF Sleeping Sickness control efforts in the Republic… – ReMIT

**Comment by zarino** [comment]
Also, is it possible to edit the page titles on the Admin side?

**Comment by stevenday** [comment]
I've updated the existing pages in the site (homepage, study page and all the user account pages) as per the list above. The other pages don't exist yet so I'm leaving the rest for now. 

When we do the search I can see how hard it will be to add that in - perhaps for the hard-coded links it will be possible, but other filters will be trickier.

It's not currently possible for the admins to edit the page titles - do you think they should be able to @zarino ?

**Comment by zarino** [comment]
Yeah, maybe we use custom titles for the "canned" searches (eg: "Studies with ERB approval pending" as shown above) but then for custom filters (eg: where the user has selected "Location: Somalia" from the sidebar) we set the title to something like "Studies matching your query" or something?

> It's not currently possible for the admins to edit the page titles

You know, I can't even remember adding that comment. I wonder why I suggested it. :confused:
